web/html/src/components/buttons.stories.tsx

```diff
@@ -1,10 +1,10 @@
-import { StoryRow, StripedStorySection } from "manager/storybook/layout";
+import { StoryRow, StorySection } from "manager/storybook/layout";
 
 import { Button } from "components/buttons";
 
 export default () => {
   return (
-    <StripedStorySection>
```

Comment:
Is there a reason to remove this here? The stripes are useful so you can see which elements have transparency and which don't.